### DISCUSSION TITLE
[v3] Unknown named parameter $fromChildren

### DISCUSSION THREAD

**Discussion by nexxai** [livewire/livewire#5886] [7. Bugs (archived)]
I'm trying to scope down the dispatching of an event to only its parent components, so as per [the docs](https://livewire.laravel.com/docs/events#listening-for-events-from-children-only), I've added the `fromChildren: true` to the On attribute for one of my methods, like this:

`#[On('item-selected', fromChildren: true)]`

The problem is that now that same page is throwing a `Unknown named parameter $fromChildren` error on load.

I did a quick search through the source of the package and the only references to `fromChildren` I can find are all in JS-related files; nothing in any PHP source.  Maybe I'm misunderstanding how PHP attributes work but this seems wrong.  My gut says there should be some kind of parameter in the `Livewire\Features\SupportEvents\On` class or something.

Can someone tell me I'm not going crazy, or (more likely) tell me what I'm doing wrong?

Thanks!

**Comment by joshhanley** [comment]
@nexxai you're not going crazy! It's documented but it seems it hasn't been implemented. Any chance you could submit a failing test PR so we can look at getting it implemented? Thanks!

**Reply by nexxai** [reply]
Hey Josh, I'm trying to write the failing test but no matter what I do, phpunit keeps failing with a `RuntimeException: Invalid path to Chromedriver [/Users/nexxai/code/livewire/vendor/laravel/dusk/src/Chrome/../../bin/chromedriver-mac-arm]. Make sure to install the Chromedriver first by running the dusk:chrome-driver command.`

I'm not clear how I should *really* install the Chromedriver since this isn't a full Laravel install.  Can you help me out here and then I'll get the PR done?

I installed the chromedriver cask through homebrew but that didn't seem to make a difference.

**Reply by PhiloNL** [reply]
This should do the trick 😃 Run from inside the livewire repository.

```shell
composer require "laravel/framework:10.*" --no-interaction --no-update --dev
composer update --prefer-stable --no-interaction
vendor/bin/dusk-updater detect --no-interaction
vendor/bin/phpunit --testsuite Browser
```

**Reply by nexxai** [reply]
Awesome, thanks.  Will get working on this right now.

**Reply by nexxai** [reply]
Thanks @PhiloNL and @joshhanley 

PR with failing test has been submitted.  I appreciate your patience and help to get me up and running.
